conf4j-core/src/main/java/com/sabre/oss/conf4j/converter/standard/BigDecimalTypeConverter.java

```diff
@@ -28,38 +28,112 @@
 
 import java.lang.reflect.Type;
 import java.math.BigDecimal;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
+import java.text.ParseException;
+import java.util.Locale;
 import java.util.Map;
 import java.util.Objects;
 
+import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 /**
  * This class converts {@link BigDecimal} to/from string.
+ * <p>
+ * The converter supports {@code format} attribute (provided in the attributes map) which specifies
```

Comment:
When _FORMAT_ static field is made constant, it can be referenced from javadoc via `{@value #FORMAT}` instead of `{@code format}`.